<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportSeedDialog</class>
 <widget class="QDialog" name="ImportSeedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Mnemonic Seed</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #282d31; color: #ddd; border: 0px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #fff;
border-right: 0px;
font-size: 20px;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Import Mnemonic Seed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Poppins</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ddd;border-right: 0px;</string>
        </property>
        <property name="text">
         <string>Recreate your wallet with the 25 word mnemonic seed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="m_errorLabel">
        <property name="styleSheet">
         <string notr="true">color: orange;</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Poppins</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ddd;border-right: 0px;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Mnemonic Seed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="m_seed">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 3px; border: 1px solid #555; border-radius: 5px;   color: #aaa;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Poppins</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ddd;border-right: 0px;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Wallet Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Poppins</family>
         </font>
        </property>
        <property name="text">
         <string>Where would you like to save your wallet?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="horizontalGroupBox_1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="m_pathEdit">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Poppins</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">padding: 3px; border: 1px solid #555; border-radius: 5px;   color: #aaa;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_selectPathButton">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Poppins</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">padding: 3px; border: 1px solid #555; border-radius: 5px;   color: #aaa;</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../../resources.qrc">
             <normaloff>:/icons/folder</normaloff>:/icons/folder</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="horizontalGroupBox_2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="b1_okButton">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Poppins</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">/* This style is handled automatically by EditableStyle.cpp */</string>
           </property>
           <property name="text">
            <string>IMPORT</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="horizontalGroupBox_3">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="b1_cancelButton">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Poppins</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">/* This style is handled automatically by EditableStyle.cpp */</string>
           </property>
           <property name="text">
            <string>CANCEL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>b1_okButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportSeedDialog</receiver>
   <slot>importButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b1_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportSeedDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_selectPathButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportSeedDialog</receiver>
   <slot>selectPathClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectPathClicked()</slot>
  <slot>importButtonClicked()</slot>
 </slots>
</ui>
